io.github.aginies.watermark.yml

```diff
@@ -0,0 +1,35 @@
+app-id: io.github.aginies.watermark
+runtime: 'org.freedesktop.Platform'
+runtime-version: '24.08'
+sdk: 'org.freedesktop.Sdk'
+command: watermark_app
+tags:
+  - utility
+  - image-processing
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --device=dri
+  - --filesystem=host
+  - --talk-name=org.freedesktop.Notifications
+  - --share=network
+modules:
+  - python3-modules.json
+  - name: watermark_app
+    no-autogen: true
+    no-make-install: true
+    buildsystem: simple
+    build-commands:
+      - tar -xvf watermark-2.3.tar.gz
+      # remove unwated files
+      - rm -rvf watermark-2.3/old watermark-2.3/windows watermark-2.3/DejaVuSans.ttf 
+      - rm -rvf watermark-2.3/Makefile watermark-2.3/test.jpg *.json
```

Comment:
```suggestion
```

Reply:
this was to cleanup, but we can remove this rm commands